pkgs/development/compilers/graalvm/oracle/default.nix

```diff
@@ -0,0 +1,12 @@
+{ stdenv
+, fetchurl
+, graalvmPackages
+, useMusl ? false
+}:
+
+graalvmPackages.buildGraalvm {
+  inherit useMusl;
+  src = fetchurl (import ./hashes.nix).hashes.${stdenv.system};
+  version = (import ./hashes.nix).version;
+  meta.platforms = builtins.attrNames (import ./hashes.nix).hashes;
```

Comment:
We need to set the proper meta here, including the license (that is probably `unfree`).

Reply:
It's actually free, but not sure what is the license

Reply:
Free to use does not mean there are no restrictions. If you know that we can redistribute it with changes to the binaries, `unfreeRedistributabe` is fine and it will allow Hydra to cache it. But if you don't, the only option here is `unfree`, and in this case there is no cache.